### PR TITLE
Components

### PR DESCRIPTION
**Pull request by den-cloud**
**For this Milestone here are the step I made to develop a React component that will hold the core functionality of a calculator.**

- Delete all the boilerplate from CRA (text, images, styles).

- Create a directory called components.

- Create a new Calculator.js file Inside components.

- In Calculator.js, create a React component that matches the design.

- Styled the calculator component.

- Add descriptive README.md file

### CODE REVIEW

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi @den-cloud, 

You've really done commendable work on this project and even went ahead of the project requirements. Great job on the tests and the entire project layout.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Inside the public directory, Kindly delete all the boilerplate from CRA (text, images, styles) as per the project requirements. Here you should only have the `index.html` file. Also, ensure that you deleted the links in the `index.html` to avoid the browser log errors.
- [ ] The `Calculator` and `App` components should be class-based components as per the project requirements. Kindly use class-based components since you will have to refactor them in the next milestone.

_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by den-cloud** [comment]
Hey, I have made all the required changes to the project & converted the Calculator functional component to a class component, but for the App component, I would say it's not necessary to do the conversion as I'm not having any state management locally. No need for a constructor, right?

File: src/App.js [hunk 1]
```diff
@@ -3,7 +3,6 @@ import Navbar from './components/Navbar';
 import Calculator from './components/Calculator';
 import Home from './components/Home';
 import Quote from './components/Quote';
-
 import './css/App.css';
 
 const App = () => (
```
Comment (TedLivist) [inline src/App.js:8]: - [ ] As highlighted by the previous reviewer, kindly convert this **`App`** component to a class-based component. As stated in the project requirements, your components should be class-based. You can add the constructor and define an empty state like so: `this.state = {}`

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @den-cloud,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
## **Hi @den-cloud ,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**